File: Week 4/DemoDistributedObject/DemoDistributedObject/ClientObjectManager.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DemoDistributedObject
{
    public class ClientObjectManager
    {
        internal static string GetObjectAttribute(int handle, string strAttributeName)
        {
            return ServerObjectManager.GetObjectAttribute(handle, strAttributeName);
        }

        internal static int CreateObject(string strClassName)
        {
            return ServerObjectManager.CreateObject(strClassName);
        }

        internal static bool SetObjectAttribute(int handle, string strAttributeName, string newValue)
        {
           return ServerObjectManager.SetObjectAttribute(handle, strAttributeName, newValue);
        }
    }
}

File: Week 4/Java code/Middleware/src/client/ClientObjectManager.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package client;

import server.ServerObjectManager;

/**
 *
 * @author chungnt
 */
public class ClientObjectManager {
    static String getObjectAttribute(int handle, String strAttributeName){
        return ServerObjectManager.getObjectAttribute(handle, strAttributeName);
    }
    
    static int createObject(String strClassName){
        return ServerObjectManager.createObject(strClassName);
    }
    
    static boolean setObjectAttribute(int handle, String strAttributeName, String newValue){
        return ServerObjectManager.setObjectAttribute(handle, strAttributeName, newValue);
    }
}
